ibis_ml/core.py

```diff
@@ -110,10 +110,13 @@ def _(X, y=None, maintain_order=False):
     # >>> X = parent[cols]
     # >>> y = parent[single_or_multiple_cols]
     # >>> table = parent[cols + single_or_multiple_cols]
+    supported_ops = tuple(
+        filter(None, (getattr(ops, name, None) for name in ["Project", "DropColumns"]))
+    )
```

Comment:
I find the following to be slightly more readable.

```suggestion
    supported_ops = tuple(
        cls for name in ["Project", "DropColumns"] if (cls := getattr(ops, name, None))
    )
```

Reply:
Agree, much better! I think I tried something with the walrus, but I put it in the beginning instead of in the if clause (so it didn't work).